public/javascripts/searchBox.js

```diff
@@ -0,0 +1,36 @@
+var enter = function(elem, checkboxClass) {
+    $(elem).keyup(function(e) {
+        if (e.keyCode == 13) {
+            $("#search-button").click()
+        }
+    })
+}
+
+
+var tag = m.prop('')
+var searchBox = {
+    view: function() {
+        return m('.input-group[style=width: 30em]', [
```

Comment:
Does this actually work for the style tag? I'm surprised, but I suppose it's okay. Alternatively you could add an argument right after this: 
`{
style: {
width: '30em'
}
}`